core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/stopstartus/StopStartUpscaleFlowConfig.java

```diff
@@ -30,9 +32,13 @@ public class StopStartUpscaleFlowConfig extends StackStatusFinalizerAbstractFlow
             new Transition.Builder<StopStartUpscaleState, StopStartUpscaleEvent>()
             .defaultFailureEvent(STOPSTART_UPSCALE_FAILURE_EVENT)
             .from(INIT_STATE)
-                    .to(STOPSTART_UPSCALE_START_INSTANCE_STATE)
+                    .to(STOPSTART_UPSCALE_GET_RECOVERY_CANDIDATES_STATE)
```

Comment:
the customers are using this? If yes, it will break the Cloudbreak

Reply:
No, it's not used by customers. Should new states be introduced differently?